### ISSUE TITLE
Missing option in delve mods

### ISSUE THREAD

**Issue by Killerninjass** [PathOfBuildingCommunity/PathOfBuilding#5889]
You cant add increased Life Reservation Efficiency of Skills under crafting items. No where do i see that they removed it is this a bug?

**Comment by Killerninjass** [comment]
So the mod still exists we just cant add it to custom items?

**Comment by QuickStick123** [comment]
Yeah you can add it via the custom items thing if you manually type it in and it will be parsed it just won't appear in the delve mod list.

**Comment by Killerninjass** [comment]
Sweet thanks for the clarification
